Bot_project/main.cpp

```diff
@@ -0,0 +1,86 @@
+#include <iostream>
+#include <string>
+
+using namespace std; //I know using namespace std is bad practice, but i'm just testing stuff out.
```

Comment:
"using namespace" in .cpp file is okay. However, it's bad if you use "using namespace" in a header file.
Having it in .cpp files does not affect other files (since no other file is including a .cpp file).
Having it in a .h file affects all files that includes that .h file. And hence, a lot of bad side effects could result.